app/models/mixins/ownership_mixin.rb

```diff
@@ -106,4 +108,8 @@ def owned_by_current_ldap_group
     ldap_group = User.current_user.try(:ldap_group)
     ldap_group && owning_ldap_group && (owning_ldap_group.downcase == ldap_group.downcase)
   end
+
+  def set_tenant_from_group
+    self.tenant_id = miq_group.tenant_id if miq_group
```

Comment:
```suggestion
    self.tenant_id = miq_group.tenant_id if miq_group_id || miq_group
```

there are rare cases where `miq_group` is set and `miq_group_id` is not set when creating the record - so I feel more comfortable having both.

But if there is an `id`, I'd prefer we use that, avoiding `MiqGroup.find(miq_group_id)`